### ISSUE TITLE
`node_modules/signal-desktop/components/long/dist/Long` does not exist

### ISSUE THREAD

**Issue by lil5** [matrix-hacks/matrix-puppet-signal#13]
```shell
> matrix-puppet-signal@1.0.0 link /home/lil5/Desktop/matrix-puppet-signal
> node -e "new (require('signal-client'))('matrix').link()"

module.js:559
    throw err;
    ^

Error: Cannot find module '/home/lil5/Desktop/matrix-puppet-signal/node_modules/signal-desktop/components/long/dist/Long'
    at Function.Module._resolveFilename (module.js:557:15)
    at Function.Module._load (module.js:484:25)
    at Module.require (module.js:606:17)
    at require (internal/module.js:11:18)
    at signalRequire (/home/lil5/Desktop/matrix-puppet-signal/node_modules/signal-client/index.js:14:35)
    at Object.<anonymous> (/home/lil5/Desktop/matrix-puppet-signal/node_modules/signal-client/index.js:78:16)
    at Module._compile (module.js:662:30)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
    at tryModuleLoad (module.js:515:12)
    at Function.Module._load (module.js:507:3)
    at Module.require (module.js:606:17)
    at require (internal/module.js:11:18)
    at [eval]:1:6
    at Script.runInThisContext (vm.js:65:33)
    at Object.runInThisContext (vm.js:197:38)

```

**Comment by lil5** [comment]
Duplicate of https://github.com/matrix-hacks/matrix-puppet-signal/issues/8

**Comment by nr23730** [comment]
Looks like an incomplete install of signal-desktop to me. 
Anyway, this should be resolved with Version 2.0 that I just uploaded.
